src/main/java/org/digitalstack/chess/Pawn.java

```diff
@@ -44,7 +46,30 @@ private void setPieceColor(PieceColor value) {
     }
 
     public void move(MovementType movementType, int newX, int newY) {
-        throw new UnsupportedOperationException("Need to implement Pawn.move()") ;
+        if(chessBoard.isLegalBoardPosition(newX, newY)) {
+            if(pieceColor==BLACK){
+                if(getXCoordinate()==newX) {
+                    if(getYCoordinate()-1==newY) {
+                        setNewCoordinates(newX,newY);
+                    }else if(getYCoordinate()==6 && (getYCoordinate()-2)==newY){
+                        setNewCoordinates(newX,newY);
+                    }
+                }
+            }else{
```

Comment:
use if(pieceColor==WHITE) instead of else, this way it's clearer what the code is doing.